src/hooks/use-event-data.js

```diff
@@ -1,35 +1,26 @@
 import { useEffect, useState } from 'react';
-
-export const removePastEvents = events => {
-    const today = new Date();
-    return events.filter(
-        e =>
-            new Date(e.node_type_attributes.event_end) >= today &&
-            e.status === 'published'
-    );
-};
+import fetchEventData from '../utils/fetch-event-data';
+import fetchLiveEventData from '../utils/fetch-live-event-data';
 
 const useEventData = url => {
     const [events, setEvents] = useState(null);
     const [error, setError] = useState(null);
     const [isLoading, setIsLoading] = useState(false);
+
     useEffect(() => {
         const getData = async () => {
             setIsLoading(true);
             try {
-                const data = await fetch(url, {
-                    credentials: 'same-origin',
-                    headers: {
-                        'Content-Type': 'application/json',
-                    },
-                });
-                if (data) {
-                    const parsedData = await data.json();
-                    const upcomingEvents = removePastEvents(parsedData);
-                    setIsLoading(false);
-                    setError(null);
-                    setEvents(upcomingEvents.reverse());
-                }
+                const eventData = await fetchEventData(url);
+                const liveData = await fetchLiveEventData();
```

Comment:
I think this is a good pattern to use in this case because this hooks primary goal is to get events data. So it makes sense for us to get both regular and live events here.